target/linux/pistachio/base-files/etc/init.d/bt-config

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+START=85
+STOP=85
+
+config_bluetooth() {
+	. /lib/functions/mac.sh
+	. /lib/functions/bluetooth.sh
+
+	local IS_BT_RUNNING
+	local timeout=5
+	local time=$(awk -F "." '{print $1}' /proc/uptime | awk '{print $1}')
+	local elap_time=0
+
+	while [ -z "$IS_BT_RUNNING" ]
+	do
+		if [ $elap_time -lt $timeout ]
+		then
+			elap_time=$( elapsed_time $time )
```

Comment:
Seeing as you need to change it anyway there is space inconsistencies here and line 30 (and a couple of others)